src/main/java/org/jenkinsci/plugins/scriptsecurity/scripts/ApprovalContext.java

```diff
@@ -90,6 +91,8 @@ public ApprovalContext withItem(@CheckForNull Item item) {
     /**
      * Gets any associated item which should be displayed to an administrator.
      */
+    // TODO: To remove, use `getActiveInstance` 1.590+ and back to `getInstance` on 1.653+
+    @SuppressFBWarnings(value = "NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE", justification = "https://github.com/jenkinsci/jenkins/pull/2094")
```

Comment:
Checked with current baseline and `-Djenkins.version=1.653` (several instances below).